File: BackEnd_ExpressJS/app/models/translation.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const Translation = sequelize.define('tblTranslation', {
        ideTranslation: {
            type: DataTypes.BIGINT,
            primaryKey: true,
        },
        strTranslation: {
            type: DataTypes.TEXT,
            allowNull: false,
            unique: true
        },
        strLabelKey: {
            type: DataTypes.STRING,
            allowNull: false,
            unique: true
        },
        createdAt: {
            type: DataTypes.DATE,
            allowNull: false,
        },
        updatedAt: {
            type: DataTypes.DATE,
            allowNull: false,
        },
    }, {
        // Activar/Desactivar las marcas de tiempo
        timestamps: true,
        // Usar el nombre de la tabla tal como está en la base de datos
        freezeTableName: true,
    });

module.exports = Translation;

File: BackEnd_Django/api/myModels/translation.py
from django.db import models

class Translation(models.Model):
    ideTranslation = models.BigAutoField(primary_key=True)
    strTranslation = models.TextField(unique=True)
    strLabelKey = models.CharField(max_length=255, unique=True)
    createdAt = models.DateTimeField(auto_now_add=True)
    updatedAt = models.DateTimeField(auto_now=True)

    class Meta:
        db_table = 'tblTranslation'
    
    def __str__(self):
        return self.strLabelKey
